src/components/Profile.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './stylesheets/profile.css';
+
+const Profile = () => {
+  const mission = useSelector((state) => state.missionReducer);
+  const rocket = useSelector((state) => state.rocketReducer);
+  return (
+    <section className="profile-container">
+      <table className="profile-mission-container">
+
+        <thead><tr><th className="table-heading"><h2 className="h2">Missions</h2></th></tr></thead>
+        <tbody className="profile-table-body">
+          {mission.map((mission) => {
+            if (mission.reserved === true) {
+              return (
+                <tr key={mission.mission_id}><td className="table-mission-name">{mission.mission_name}</td></tr>
+              );
+            }
+            return null;
+          })}
+        </tbody>
+      </table>
+      <table className="profile-rocket-container">
+        <thead><tr><th className="table-heading"><h2 className="h2">Rockets</h2></th></tr></thead>
+        <tbody>
+          {rocket.map((rocket) => {
+            if (rocket.reserved === true) {
+              return (
+                <tr key={rocket.rocket_id}><td className="table-rocket-name">{rocket.rocket_name}</td></tr>
```

Comment:
## [OPTIONAL]

- [ ] kindly put some padding around this test in your table.

![image](https://user-images.githubusercontent.com/65068771/156462146-eea41cd5-f09e-425c-8c04-52f6eb758395.png)